src/components/pages/auth/register/index.tsx

```diff
@@ -130,21 +134,44 @@ export default function RegisterComponent() {
               >
                 Password <span className="text-red-600">*</span>
               </label>
-              <input
-                type="password"
-                name="password"
-                id="password"
-                required={true}
-                onChange={changeHandler}
-                placeholder="Enter your password"
-                className="w-full rounded-md bg-white py-2 px-4 text-sm md:text-base font-medium text-secondary outline-none border border-white focus:border-secondary-light dark:border-secondary-light dark:focus:border-white"
-              />
+              <div className="relative">
+                <input
+                  type={showPassword ? "text" : "password"}
+                  name="password"
+                  id="password"
+                  required={true}
+                  onChange={changeHandler}
+                  placeholder="Enter your password"
+                  className="w-full rounded-md bg-white py-2 px-4 text-sm md:text-base font-medium text-secondary outline-none border border-white focus:border-secondary-light dark:border-secondary-light dark:focus:border-white"
+                />
+                <div>
+                  <button
+                    className="absolute top-1/2 right-2 transform -translate-y-1/2 cursor-pointer"
+                    onClick={(e) => {
+                      setShowPassword(!showPassword);
+                      e.stopPropagation();
```

Comment:
What is `e.stopPropagation();`? What is the use of this?

Reply:
removed this line.